src/migrations/20231107000000-ActivityReportGoalFieldResponse-incedint.js

```diff
@@ -0,0 +1,67 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+
+      await queryInterface.sequelize.query(/* sql */`
+        WITH
+          incidents AS (
+            SELECT
+              dml_txid,
+              dml_by,
+              dml_type,
+              COUNT(*),
+              MIN(dml_timestamp),
+              MAX(dml_timestamp)
+            FROM "ZALActivityReportGoalFieldResponses" zargfr
+            GROUP BY 1,2,3
+            HAVING COUNT(*) > 400
```

Comment:
I get that your flagging incidents in this CTE. But why not just simply take everything that appears more than once with the same txn id as an incident. Why do we say exactly 400. Can we have the same txn id over mutliple rows?

Reply:
Or the inverse of this and get the max id (the value to set).

Reply:
NVM I see what you are saying here we need to catch updates to all rows as "bad".